tests/yam/agama/boot_agama.pm

```diff
@@ -87,7 +87,11 @@ sub run {
     $grub_entry_edition->move_cursor_to_end_of_kernel_line();
     $grub_entry_edition->type(\@params);
     $grub_entry_edition->boot();
-    $agama_up_an_running->expect_is_shown();
+    if (get_var('EXTRABOOTPARAMS', '') =~ /systemd.unit=multi-user.target/) {
```

Comment:
could you please add a blank line to separate this block? looks a bit ugly but we will refactor this.